### PR TITLE
Build an OTP application release.

### PR DESCRIPTION
**Pull request by cmeiklejohn**
@kellymclaughlin @reiddraper @densone

### CODE REVIEW

File: rel/vars.config [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+{platform_data_dir, "./data"}.
+{platform_etc_dir,  "./etc"}.
+{platform_lib_dir,  "./lib"}.
+{platform_log_dir,  "./log"}.
+
+%%
+%% etc/app.config
+%%
+{cs_hostname, "s3.amazonaws.com"}.
+{cs_port, 8080}.
```
Comment (kellymclaughlin) [inline rel/vars.config]: The default here should be `80`

**Comment by cmeiklejohn** [comment]
Any other concerns here, @kellymclaughlin ?

File: rel/vars.config (hunk 1 above)
Comment (cmeiklejohn) [inline rel/vars.config]: Fixed in [788d683f313e61f319f0b2de0a07f36d7dd5cf8c].

**Comment by kellymclaughlin** [comment]
I got this error when running `make stage`:

```
==> rel (generate)
ERROR: Unable to generate spec: read file info /Users/kelly/basho/repos/riak_cs_control/rel/../apps/riak_cs_control/include failed
make: *** [rel] Error 1
```

There is no toplevel `include` directory so you need to remove `apps/riak_cs_control/include`.

**Comment by cmeiklejohn** [comment]
Good call, removed.  Weird this didn't pose a problem for me with R15B02.

**Comment by kellymclaughlin** [comment]
The startup is very noisy. I'd recommend adding this to `rel/files/app.config`:

```
{lager, [
          %% What handlers to install with what arguments                                                                                                                                                                                                                                                                                                                   
          %% The defaults for the logfiles are to rotate the files when                                                                                                                                                                                                                                                                                                     
          %% they reach 10Mb or at midnight, whichever comes first, and keep                                                                                                                                                                                                                                                                                                
          %% the last 5 rotations. See the lager README for a description of                                                                                                                                                                                                                                                                                                
          %% the time rotation format:                                                                                                                                                                                                                                                                                                                                      
          %% https://github.com/basho/lager/blob/master/README.org                                                                                                                                                                                                                                                                                                          
          %%                                                                                                                                                                                                                                                                                                                                                                
          %% If you wish to disable rotation, you can either set the size to 0                                                                                                                                                                                                                                                                                              
          %% and the rotation time to "", or instead specify a 2-tuple that only                                                                                                                                                                                                                                                                                            
          %% consists of {Logfile, Level}.                                                                                                                                                                                                                                                                                                                                  
          {handlers, [
                      {lager_console_backend, info},
                      {lager_file_backend, [
                                            {"{{platform_log_dir}}/error.log", error, 10485760, "$D0", 5},
                                            {"{{platform_log_dir}}/console.log", info, 10485760, "$D0", 5}
                                           ]}
                     ]},

          %% Whether to write a crash log, and where.                                                                                                                                                                                                                                                                                                                       
          %% Commented/omitted/undefined means no crash logger.                                                                                                                                                                                                                                                                                                             
          {crash_log, "{{platform_log_dir}}/crash.log"},

          %% Maximum size in bytes of events in the crash log - defaults to 65536                                                                                                                                                                                                                                                                                           
          {crash_log_msg_size, 65536},

          %% Maximum size of the crash log in bytes, before its rotated, set                                                                                                                                                                                                                                                                                                
          %% to 0 to disable rotation - default is 0                                                                                                                                                                                                                                                                                                                        
          {crash_log_size, 10485760},

          %% What time to rotate the crash log - default is no time                                                                                                                                                                                                                                                                                                         
          %% rotation. See the lager README for a description of this format:                                                                                                                                                                                                                                                                                                         %% https://github.com/basho/lager/blob/master/README.org                                                                                                                                                                                                                                                                                                          
          {crash_log_date, "$D0"},

          %% Number of rotated crash logs to keep, 0 means keep only the                                                                                                                                                                                                                                                                                                    
          %% current one - default is 0                                                                                                                                                                                                                                                                                                                                     
          {crash_log_count, 5},

          %% Whether to redirect error_logger messages into lager - defaults to true                                                                                                                                                                                                                                                                                                  {error_logger_redirect, true}
         ]},

 %% SASL config                                                                                                                                                                                                                                                                                                                                                             
 {sasl, [
         {sasl_error_logger, false}
        ]}
```

**Comment by cmeiklejohn** [comment]
Addressed in [a6eba56].

**Comment by kellymclaughlin** [comment]
My only other comment is it might be nice to add a `lager:info` when the webmachine interface starts up to tell which port the app is accepting connections on.

**Comment by cmeiklejohn** [comment]
@kellymclaughlin see [48fee807ad405157eb940ba3518e5053ae67ca6a].

**Comment by kellymclaughlin** [comment]
+1 to merge

**Comment by cmeiklejohn** [comment]
Sweet, thank you!

**Pull request merged**